be/src/util/mem_info.cpp

```diff
@@ -127,6 +133,18 @@ void MemInfo::process_full_gc() {
         return;
     }
     freed_mem += MemTrackerLimiter::free_top_memory_query(_s_process_full_gc_size - freed_mem);
+    if (freed_mem > _s_process_full_gc_size) {
+        return;
+    }
+    if (config::enable_query_memroy_overcommit) {
+        freed_mem += MemTrackerLimiter::free_top_overcommit_query(
+                _s_process_full_gc_size - freed_mem, MemTrackerLimiter::Type::LOAD);
+        if (freed_mem > _s_process_full_gc_size) {
+            return;
+        }
+    }
+    freed_mem += MemTrackerLimiter::free_top_memory_query(_s_process_full_gc_size - freed_mem,
```

Comment:
I think this should be put in `if (config::enable_query_memroy_overcommit)`?
Otherwise, if `enable_query_memroy_overcommit` is false, this will cancel the load prior to query?

Reply:
![image](https://user-images.githubusercontent.com/13197424/215236867-9eb21fd6-3e64-48b4-b4b4-fab7d52a1849.png)
This problem will not exist, because cancel query first, then cancel load.
And first cancel top overcommit, then cancel top meory.
cancel top overcommit query in minor gc.
full gc:
```
// step1: free all cache
// step2: free top memory query
// step3: free top overcommit load
// step4: free top memory load
```

Reply:
done